io/net/bonding.py

```diff
@@ -285,9 +296,13 @@ def bond_fail(self, arg1):
                 self.fail("Not able to bring up the slave\
                                 interface %s" % interface)
             time.sleep(self.sleep_time)
-
-        bond_mtu = ['2000', '3000', '4000', '5000', '6000', '7000',
-                    '8000', '9000']
+        
+        bond_mtu = []
+        if self.is_vnic():
+            bond_mtu = ['9000']
+        else:
+            bond_mtu = ['2000', '3000', '4000', '5000', '6000', '7000',
+                        '8000', '9000']
```

Comment:
@vaishnavibhat we could avoid else by

```
        bond_mtu = ['2000', '3000', '4000', '5000', '6000', '7000',
                        '8000', '9000']
        if self.is_vnic():
            bond_mtu = bond_mtu[-1]
```

Reply:
@vaishnavibhat Why not move is_vnic to utils ? this will be used in other tests too ?